docs/Utils.md

```diff
@@ -0,0 +1,34 @@
+# Collection of helpful utilities
+
+Here you can find a library of utilities that are used often within your application. This includes hooks, helper functions, and various tools.
```

Comment:
I miss the humor we used to inject in these things.

```suggestion
Here you can find a library of utilities that are used often within your application next to the stapler, tape, scissors and migraine meds. This includes hooks, helper functions, and that thing from that wedding.
```

(Don't accept this...lol)